src/features/drop-manager/routes/ticket/[id].tsx

```diff
@@ -65,7 +55,7 @@ export default function TicketDropManagerPage() {
     getWallet();
   }, [selector]);
 
-  const { masterKeyValidity } = useValidMasterKey({ dropId });
+  const { masterKeyValidity } = useValidMasterKey({ dropId: dropId as string });
```

Comment:
not related to the mission of this PR but if dropId is not part of the params for some reason, we should redirect it to All Drops or go to 404